### ISSUE TITLE
Support reading version 2 npys

### ISSUE THREAD

**Issue by MRedies** [fhs/NPZ.jl#31]
If I want to open an *.npy file saved in version 2 format (https://docs.scipy.org/doc/numpy-1.14.0/neps/npy-format.html) I get:
```
julia> den = npzread("bla.npy")
ERROR: unsupported NPZ version
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] npzreadarray(::IOStream) at ~/.julia/packages/NPZ/TnTz7/src/NPZ.jl:209
 [3] npzread(::String) at ~/.julia/packages/NPZ/TnTz7/src/NPZ.jl:245
 [4] top-level scope at none:0
```
I belive adding support for v2 is quite easy:
Line 211
```julia
hdrlen = readle(f, UInt16)
```
has to be replaced with an if-statement, that executes
```julia
hdrlen = readle(f, UInt32)
```
in case of v2.

**Comment by MRedies** [comment]
I tried to fix it in https://github.com/fhs/NPZ.jl/pull/32

**Comment by fhs** [comment]
Closing because fixed in #32.
